CONTRIBUTING.md

```diff
@@ -43,7 +45,7 @@ When reporting issues, always include the output of `go env`.
 
 Also include the steps required to reproduce the problem if possible and
 applicable. This information will help us review and fix your issue faster.
-When sending lengthy log-files, consider posting them as a gist [https://gist.github.com](https://gist.github.com).
+When sending lengthy log-files, consider posting them as a [gist](https://gist.github.com).
```

Comment:
This fixes syntax highlighting in Vim. If the full URL (including `https://`) is actually desired in the text, I believe markdown will always render it as a link automatically even without the `[]()` syntax:

```suggestion
When sending lengthy log-files, consider posting them as a https://gist.github.com.
```